src/env-inl.h

```diff
@@ -256,8 +256,12 @@ inline void AsyncHooks::RemoveContext(v8::Local<v8::Context> ctx) {
   v8::Isolate* isolate = env()->isolate();
   v8::HandleScope handle_scope(isolate);
   for (auto it = contexts_.begin(); it != contexts_.end(); it++) {
+    if (it->IsEmpty()) {
+      contexts_.erase(it);
+      break;
```

Comment:
why break?

Reply:
Yeah that should be a continue. Fixing that now.